wolfssl/openssl/ssl.h

```diff
@@ -761,7 +772,11 @@ wolfSSL_X509_STORE_set_verify_cb((WOLFSSL_X509_STORE *)(s), (WOLFSSL_X509_STORE_
 #define ASN1_STRING_free                wolfSSL_ASN1_STRING_free
 #define ASN1_STRING_cmp                 wolfSSL_ASN1_STRING_cmp
 #define ASN1_STRING_data                wolfSSL_ASN1_STRING_data
+#if defined(WOLFSSL_QT)
+#define ASN1_STRING_get0_data           wolfSSL_ASN1_STRING_get0_data
```

Comment:
Is there a reason we don't expose and use wolfSSL_ASN1_STRING_get0_data() for all OPENSSL_EXTRA, not just WOLFSSL_QT?